infra/feast-operator/test/e2e_rhoai/resources/feast-wb-connection-credit-scoring.ipynb

```
@@ -1,449 +1,633 @@
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# validate_list_data_sources for with and without permissions \n",
+        "\n",
+        "import os\n",
+        "from feast.errors import FeastPermissionError\n",
+        "\n",
+        "def validate_list_data_sources(fs_obj):\n",
+        "    \"\"\"\n",
+        "    Validates list_data_sources() with special handling for Kubernetes auth mode.\n",
+        "    If CONFIGMAP_DATA indicates auth=kubernetes, expect FeastPermissionError.\n",
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)